index.html

```diff
@@ -352,5 +332,83 @@ <h2 class="text-center">Skills</h2>
         </div>
       </div>
     </section>
+
+    <section class="contact-section relative">
+      <div class="contact-header">
+        <h2 class="text-center">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+      <div class="contact-form">
+        <form action="https://formspree.io/f/xgebdabq" method="POST" class="flex flex-col">
+          <input
+            maxlength="30"
+            class="contact-input text-dark"
+            name="name"
+            type="text"
+            placeholder="Full name"
+            required
+          />
+
+          <input
+            maxlength="30"
```

Comment:
- [ ] Kindly remove the max length validation that you added to your email input which is not a requirement for this project. 

Note: some users can have long emails 😄